smartmeter_datacollector/smartmeter/iskraam550.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, port: str) -> None:
             fallback_id=port,
             register_obis=ISKRA_AM550_COSEM_REGISTERS
         )
-        self._parser = HdlcDlmsParser(cosem_config)
+        self._parser = HdlcDlmsParser(cosem_config, decryption_key)
```

Comment:
Should we add a log warning that this is untested?

Reply:
done! good idea!